test-vectors/json/zip_0320.json

```diff
@@ -0,0 +1,19 @@
+[
+    ["From https://github.com/zcash-hackworks/zcash-test-vectors/blob/master/transparent/zip_0320.py"],
```

Comment:
This URL is not correct, it should be https://github.com/zcash/zcash-test-vectors/blob/master/zcash_test_vectors/transparent/zip_0320.py